pipeline/scripts/5/0/tier-3/test-rados-mondb-trimming.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/sh
+echo "Beginning Ceph RADOS regression testing for Mon DB trimming"
+
+random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
+instance_name="ci-${random_string}"
+platform="rhel-8"
+rhbuild="5.0"
+test_suite="suites/pacific/rados/tier-3_rados_mondb_trimming.yaml"
```

Comment:
Would it make sense to rename the file as `tier-3_raods_test-mon-db-trimming.yaml`?

Reply:
Renamed the suite file